apps/src/lib/levelbuilder/lesson-editor/FindResourceDialog.jsx

```diff
@@ -35,6 +35,17 @@ class FindResourceDialog extends Component {
     this.setState({selectedResourceKey: e.target.value});
   };
 
+  formatResourceName = resource => {
+    let formattedName = resource.name;
+    if (resource.properties.type) {
+      formattedName += ` - ${resource.properties.type}`;
+    }
+    if (resource.properties.audience) {
```

Comment:
Did they request to have it include audience too? I think it was just type

Reply:
No, just type was brought up I think. Do you think I should remove audience? I don't have an opinion either way really -- just thought it might be easier to see audience too

Reply:
I think audience makes it long and I don't think its that helpful in this context so I would leave it off until we get asks for it

Reply:
okay -- will change. Thanks!